# 예외처리
- 예외가 발생하는 부분 -> try {}
- 예외를 처리하는 부분 -> catch() {}

```kotlin

 import java.lang.Exception
 try {
    // 코드 -> 예외 발생 가능한 코드
} catch(e: Exception) {
    // 예외를 처리하는 코드
} finally {
    // 마지막에 반드시 거치는 작업
}
```

- try문에서 A라는 예외가 발생하면 catch문에서 A라는 예외를 처리하겠고 명시적으로 적어줘야 한다. catch하지 않는 예외가 있다면 프로그램이 종료된다.
- 모든 종류의 예외를 포함하는 예외 타입 -> Exception
- try, catch는 독립적으로 사용할 수 없다.₩